src/index.js

```diff
@@ -125,6 +126,41 @@ class Mongo {
     }
   }
 
+  async generateVerificationEmailToken(userId: string, email: string): Promise<void> {
```

Comment:
The majority of the logic should occur in the core accounts package. The database packages should handle the bare minimum (writes and reads).